templates/home/home-en.html

```diff
@@ -165,14 +165,14 @@ <h2>Government initiatives</h2>
 	<div class="row wb-eqht">
 		<div class="col-sm-6">
 			<div class="well well-sm brdr-rds-0 eqht-trgt">
-				<img class="img-responsive full-width" src="../img/520x200.png" alt=""/>
+				<img class="img-responsive full-width" src="../../components/gc-features/img/initiative-520x200.png" alt=""/>
```

Comment:
Here's a trickier situation...

I made the home page point to a feature image under ``/components/gc-features/img/``. Why? Because at heart it's a GC feature image and the home page just so happens to be using it.

I could've placed the ``initiative-520x200.png`` image under a top-level ``/img/`` folder or ``/components/img/``, but felt like going in that direction would make it a nightmare to manage images going forward.

For instance, if some random template started using a pre-existing component's image in the future, I don't think anyone would want to be stuck having to move the image around in the repo to a higher-level folder to fit the new reality of multiple "things" using that image for the same purpose.